src/NuGet.Services.Staging.Database/Models/StageContext.cs

```diff
@@ -15,6 +16,8 @@ public class StageContext : DbContext
 
         public virtual DbSet<StageMembership> StageMemberships { get; set; }
 
+        public virtual DbSet<PackageMetadata> PackagesMetadata { get; set; } 

```

Comment:
I will leave it

---
In reply to: [64698881](https://github.com/NuGet/NuGet.Staging/pull/50#discussion_r64698881) [](ancestors = 64698881)